packit/config/aliases.py

```diff
@@ -175,6 +177,145 @@ def get_branches(
     return branches
 
 
+def get_all_fast_forward_branches(
```

Comment:
It was needed with my previous implementation, now it isn't. But since it tooks me sometime to make them and are just helper functions I thought that I can keep them if they can ever be handy.

Reply:
I think I would rather see a function that would take `dist_git_branches` and expand both keys and values (values first, to allow sharing them between different keys). Then all 3 functions could have just a few lines of code, and could be implemented on demand, if the need arises.

Reply:
It is a good refactoring but since we don't have the need of this function I will remove them. I don't think is worth spending time on them anymore.